draft-ietf-quic-qlog-main-schema.md

```diff
@@ -48,53 +48,63 @@ informative:
-called qlog.  This format allows easy sharing of data and the creation of reusable
-visualization and debugging tools. The high-level schema in this document is
-intended to be protocol-agnostic. Separate documents specify how the format should
-be used for specific protocol data. The schema is also format-agnostic, and can be
-represented in for example JSON, csv or protobuf.
+This document defines qlog, an extensible high-level schema for a standardized
+logging format. It allows easy sharing of data, benefitting common debug and
+analysis methods and tooling. The high-level schema is independent of protocol;
+separate documents extend qlog for protocol-specific data. The schema is also
+independent of serialization format, allowing logs to be represented in many
+ways such as JSON, CSV, or protobuf.
+
+--- note_Note_to_Readers
+
+> Note to RFC editor: Please remove this section before publication.
+
+Feedback and discussion are welcome at
+[https://github.com/quicwg/qlog](https://github.com/quicwg/qlog). Readers are
+advised to refer to the "editor's draft" at that URL for an up-to-date version
+of this document.
+
+Concrete examples of integrations of this schema in
+various programming languages can be found at
+[https://github.com/quiclog/qlog/](https://github.com/quiclog/qlog/).
 
 --- middle
 
 # Introduction
 
-There is currently a lack of an easily usable, standardized endpoint logging
-format. Especially for the use case of debugging and evaluating modern Web
-protocols and their performance, it is often difficult to obtain structured logs
-that provide adequate information for tasks like problem root cause analysis.
-
-This document aims to provide a high-level schema and harness that describes the
-general layout of an easily usable, shareable, aggregatable and structured logging
-format. This high-level schema is protocol agnostic, with logging entries for
-specific protocols and use cases being defined in other documents (see for example
-{{QLOG-QUIC}} for QUIC and {{QLOG-H3}} for HTTP/3 and QPACK-related event
-definitions).
+Endpoint logging is a useful strategy for capturing and understanding how
+applications using network protocols are behaving, particularly where protocols
+have an encrypted wire image that restricts observers ability to see what is
+happening.
+
+Many applications implement logging but the format tends to be custom rather than
+standardized. This has a effect on the tools and methods that are used to
+analyze the logs, for example to perform root cause analysis of an
+interoperability failure between distinct implementations. A lack of a common
+format impedes the development of common tooling that can be used by all parties
+that have access to logs.
+
+This document defines qlog, an extensible high-level schema and harness that
+provides a shareable, aggregatable and structured logging format intended to be
+more usable. This high-level schema is independent of protocol, with logging
```

Comment:
```suggestion
provides a shareable, aggregatable and structured logging format. This high-level schema is independent of protocol, with logging
```

"more usable" is vague... more usable than what? I propose just cutting this

Reply:
me too!